src/Modals/ModalPhotoDisplay/ModalPhotoDisplay.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import "./ModalPhotoDisplay.css";
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faAngleLeft, faAngleRight} from '@fortawesome/free-solid-svg-icons'
+import { connect } from 'react-redux';
+import Actions from "../../Actions/actions";
+library.add(faAngleLeft, faAngleRight) 
+
+class ModalPhotoDisplay extends Component {
+    isCurrentClikedImage(image) {
+        return image === this.props.currentClickedImage
+    }
+
+    renderThumbs() {
+        let images = [];
+        for (let image of this.props.images) {
+            images.push(
+                <img src={image} 
+                        alt="" key={image} 
+                        className={`thumbs ${(this.isCurrentClikedImage(image)) ? 'currentClickedImageThumb' : ''}`}
+                        onClick={ ()=> this.props.updateCurrentImage(image)}>
+                </img>
+            )
+        }
+        return (
+            <div className="thumbHolder">
+                {images}
+            </div>
+        )
+    }
+
+    handleClickedArrow(isLeft){
+        const currentImageIndex = this.props.images.indexOf(this.props.currentClickedImage);
+        let newIndex = -3;
```

Comment:
It's really confusing why anything would start with `-3`. These are called [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)), and should be avoided.